qc_loop.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+export RUST_BACKTRACE=1
+
+while true
+do
+    cargo test qc_ --release -- --nocapture
+    if [[ x$? != x0 ]] ; then
+        exit $?
+    fi
+done
```

Comment:
I don't understand the purpose of the quick_check here 🤔

Reply:
I have added some tests with quickcheck and those tests are prefixed by `qc_`, this script runs them in a loop as the inputs are randomized. I found one bug when the reader cursor try to read an empty file.